### ISSUE TITLE
Install error - vscode-dts/vscode.proposed.inlineCompletions.d.ts not found

### ISSUE THREAD

**Issue by sr57** [CodedotAl/code-clippy-vscode#14]
npm install

```
...
From: https://raw.githubusercontent.com/microsoft/vscode/main/src/vscode-dts/vscode.proposed.inlineCompletions.d.ts
Read more about proposed API at: https://code.visualstudio.com/api/advanced-topics/using-proposed-api
Failed to get https://raw.githubusercontent.com/microsoft/vscode/main/src/vscode-dts/vscode.proposed.inlineCompletions.d.ts
...
```

**Comment by codetin** [comment]
![image](https://user-images.githubusercontent.com/203521/179891236-7112368a-2bd6-416b-a0a2-60400439a744.png)
now there are just another 2 file

**Comment by ncoop57** [comment]
Please try now with the updated version that no longer requires VSCode insider and see if this error goes away

**Comment by SeeNoBee** [comment]
Hello. I can confirm that the issue is still there.

**Comment by create-juicey-app** [comment]
issue still here.

**Comment by ncoop57** [comment]
Fixed in #1b1edf6ccbb2568cb1873459f53d63f5b6dbce2e
